src/backend/src/utils/slack.utils.ts

```diff
@@ -396,10 +396,18 @@ export const sendDRScheduledSlackNotif = async (
   }
 };
 
-export const sendSlackCRReviewedNotification = async (slackId: string, crId: string, identifier: number) => {
+export const sendSlackCRReviewedNotification = async (
+  slackId: string,
+  crId: string,
+  identifier: number,
+  comments: string | null
+) => {
   if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
   const msgs = [];
-  const fullMsg = `:tada: Your Change Request was just reviewed! Click the link to view! :tada:`;
+  // let fullMsg;
+  const fullMsg = !comments
```

Comment:
The ternary can be inline in the string because in this case there is still repetition of the string when there is and isn't comments. For example `"...reviewed!\n${comments ? 'some string with the comments' : ''}..."`